cmd/svcat/browsing/marketplace_cmd.go

```diff
@@ -66,7 +66,7 @@ func (c *MarketplaceCmd) Run() error {
 		Namespace: c.Namespace,
 		Scope:     servicecatalog.AllScope,
 	}
-	classes, err := c.App.RetrieveClasses(opts)
+	classes, err := c.App.RetrieveClasses(opts, "")
```

Comment:
I need to say that I do not like such API which requires from you to set some values when you are not interested in customizing that. IMO the best option here will be the _functional options pattern_. As a result you will have 
```go
# default behaviour
c.App.RetrieveClasses()

# sth changed
c.App.RetrieveClasses(WithBrokerName("myqsl"), InNamespace("system"))
```

benefits:
- in future you do not need to break the function signature (always backward compatible) (and this is already named as SDK so it is used by others)
- only can change the default behavior only if want without setting some additional values


but I joined the SC after the svcat was introduced so maybe let's do not do such huge refactor as **you already did a nice work 👍** 

but at least let's try to avoid the magic value and introduce const for that case, similar approach is used in k8s for empty string in clients:
https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L287-L290

so sth like `c.App.RetrievClasses(opts, FromAllBrokers)`